aFWall/src/main/java/dev/ukanth/ufirewall/util/G.java

```diff
@@ -97,6 +97,7 @@
 	private static final String ISKINGDETECT = "kingDetect";
 	private static final String PWD_ENCRYPT= "pwdEncrypt";
 	private static final String PROFILE_PWD= "profilePwd";
+	private static final String FINGERPRINT_ENABLED = "fingerprintEnabled";
```

Comment:
Please ignore the values on backup/restore functionality. Have a look at 
https://github.com/vzool/afwall/blob/8da8a572eefabcb849a1d2756dda9c90892eb9c7/aFWall/src/main/java/dev/ukanth/ufirewall/Api.java#L2402